### ISSUE TITLE
Missing fruits

### ISSUE THREAD

**Issue by rotsee** [fruktkartan/fruktkartan#40]
A collection of things people have asked for, that we could consider including:

- [ ] Jordgubbar (Strawberries)
- [X] Körsbärskornell (Cornelian cherry)
- [ ] Blåhallon/salmbär (Dewberry)
- [ ] Jordärtskocka (Jerusalem artichoke)
- [ ] Various herbs. We probably need some kind of policy on whether to includes edible things that are not fruits!
- [X] Körsbärsplommon (cherry plum)
- [ ] Rönnbär (rowan berries)
- [ ] Ramslök (wild garlic)
- [ ] Krusbär (gooseberry)
- [ ] bärhäggmispel (saskatoon)

Please leave a comment if you'd like to add something to the list!

**Comment by JoachimVem** [comment]
Heres a list of suggesitons (Quite alot of them so feel free to ignore many of them)

Blåhallon/Salmbär
Fläder
Humle
Glasört
Kungsmynta
Saltarv
Skogslök

**Comment by stemid** [comment]
- [ ] Nypon

**Comment by quite** [comment]
@stemid Nypon (rosehips) could already be added to the map. It will now also be possible to filter on them, in the dropdown (prod should be deployed in a few minutes).

**Comment by rotsee** [comment]
Quick note on nypon: I did not include them when migrating the old database, arguing that there are shrubs practically _everywhere_, and they are of much less interest to most people that fruit trees anyway. If we have decided to include them again, I could migrate a couple of hundred ”trees” from the old site.

But this, again, brings us to the question about **scope**: What do we want to include? I do not have any strong opinions about rosehips, but one thing I would like to avoid the fate of Falling Fruits and other sites, where even e.g. linden trees are added (you can make tea etc from them), making the maps very, very noisy, imo. Again, this depends on the intended target audience. But if we want the site to work as inspiration and an eye opener (“what can I find where I live?”), I'd suggest focusing on apple trees rather than linden trees. Including everything edible in nature (and most things are, in some quantity...) might be useful for power users on the other hand, but will require more active filtering, and already knowing what to look for.

**Comment by quite** [comment]
@rotsee very good points, have not thought so much about it. I've seen some other sites, but not fallingfruit.org before, it's crammed!

People have been adding a few rosehips during the past months, so I guess there is no harm in allowing to filter on them. But I should discuss things a bit more in advance next time :)

**Comment by stemid** [comment]
Nypon are a staple in Swedish cuisine and have a lot of great uses. The reason I bring it up is actually because someone on Reddit asked where they could find Nypon bushes. I was going to recommend fruktkartan.se but noticed there were no Nypon on it.

**Comment by rotsee** [comment]
User request via mail: rowan (rönn)

**Comment by rotsee** [comment]
Mejlönskemål: Ullungrönn

**Comment by rotsee** [comment]
Mejlönskemål: Ginkgo (hur vanligt är det att de ger frukt i Sverige?)

**Comment by quite** [comment]
Inte här nere iaf vad jag sett.

**Comment by quite** [comment]
Should the fruit svgs be present in booth `public/markers/` and in `app/assets/icons/`? They seem to be, but also files are not identical (like berry-blackcurrant, the only one i checked)

**Comment by quite** [comment]
> Should the fruit svgs be present in booth `public/markers/` and in `app/assets/icons/`? They seem to be, but also files are not identical (like berry-blackcurrant, the only one i checked)

They should. Now we have gooseberries! Commit https://github.com/fruktkartan/fruktkartan/commit/e58b8264ceaaa222e24ad73cf64de82308e5c303 might be helpful for seeing how a new berry was added.

**Comment by ajain-93** [comment]
Fantastisk karta.
Jag noterar att `blåbär` finns i [denna lista](https://github.com/fruktkartan/fruktkartan/blob/360082a509fba62f152afbabc777f126314f6312/server/utils/treeGroupMap.ts#L3), men inte i övrigt på kartan.

**Comment by quite** [comment]
> Fantastisk karta. Jag noterar att `blåbär` finns i [denna lista](https://github.com/fruktkartan/fruktkartan/blob/360082a509fba62f152afbabc777f126314f6312/server/utils/treeGroupMap.ts#L3), men inte i övrigt på kartan.

Yeah, treeGroupMap.ts is only (i think) used to map tree/plant to a group. But I added Blåbär/Blueberry in insertableTrees.json now, so it should be possible to actually add it to the map.

**Comment by rotsee** [comment]
Maybe it's time to revisit this discussion again: https://github.com/fruktkartan/fruktkartan/issues/40#issuecomment-701256972

I am sceptical to allowing things like buleberries here, as the number of “trees“ could grow until the map (as it currently work) would be unusable. My argument is that while power users would probebly be able to use a more sophisticated filter function to make use of the map, a for a casual user a map where virtually every tree and brushery is marked doesn't really help at all (I've seen this happen on some similar global and US sites).

One way out of this dilemma could be to decide that the scope for Fruktkartan is fruits, at the very least when it comes to adding to types.

**Comment by quite** [comment]
@rotsee I did not remember the discussion, and did not think much, but I agree. I'll revert blueberry for now.
